setup.py

```diff
@@ -27,6 +27,7 @@
         'numpy>=1.7',
         'fasteners',
         'numcodecs>=0.5.3',
+        "pyosreplace : python_version < '3.3' and sys.platform == 'win32'",
```

Comment:
Some examples show this as being a semicolon instead.

Reply:
Ah, that explains why the env markers didn't work. If nobody complains I'll leave the later commit.